### DISCUSSION TITLE
[@types/superagent]: Cannot find name 'Blob'

### DISCUSSION THREAD

**Discussion by strmer15** [DefinitelyTyped/DefinitelyTyped#58124] [Issues with a @types package]
After commit https://github.com/DefinitelyTyped/DefinitelyTyped/commit/a896b3dca5821599ad51d9e795ba2d491a45953e, @types/superagent no longer includes the `dom` typings, so `Blob` can no longer be found. Now my package that uses it via @types/supertest fails to compile with:

```
$ tsc --build
../../node_modules/@types/superagent/index.d.ts:34:29 - error TS2304: Cannot find name 'Blob'.

34 type MultipartValueSingle = Blob | Buffer | fs.ReadStream | string | boolean | number;
                               ~~~~
```

[a comment by an automated account is not shown]

**Reply by carnesen** [reply]
@strmer15 Updating your `@types/node` dependency should fix it. https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/55382#discussioncomment-1890788

**Reply by strmer15** [reply]
@carnesen thanks, but we're still on Node 12, so can't do that yet. I really don't think that a patch release should be breaking everyone who's still on Node 12.

**Reply by alecgibson** [reply]
@strmer15 sorry about the break! Until you bump to Node.js v14+, I'd recommend one of the following:

 - pin your `@types/superagent` version
 - explicitly add `dom` to your `tsconfig.json` typings (this is basically what `@types/superagent` was doing before anyway, but now it's made clearer that you need the `dom` typings)
